src/content/self-hosted/eks.mdx

```diff
@@ -258,7 +241,7 @@ NAME                             TYPE           CLUSTER-IP   EXTERNAL-IP
 okteto-ingress-nginx-controller  LoadBalancer   10.0.7.73    a519c8b3b27f94...elb.amazonaws.com   80:30795/TCP,443:32481/TCP,1234:30885/TCP   5m
 ```
 
-You'll need to take the `EXTERNAL-IP` address, and add it to your DNS for the domain you have chosen to use. In Route 53, this is done by creating an `A` record with the name  `*`, pointing to the alias of the Elastic Load Balancer.
+You'll need to take the `EXTERNAL-IP` address, and add it to your DNS for the domain you have chosen to use. In Route 53, this is done by creating an `A` record with the name `*`, pointing to the alias of the Elastic Load Balancer.
```

Comment:
in some parts we call it subdomain, in others we call it domain. we should be consistent.